_includes/devdoc/guide_dash_features.md

```diff
@@ -730,6 +730,24 @@ voted to delete them.
 Sentinel manages superblock creation, voting, and submission to dashd for
 network propagation.
 
+Beginning ~3 days (1662) blocks prior to a superblock, Sentinel selects one
+masternode per block to rank proposals. This ranking is used to determine
+what a candidate superblock (or "superblock trigger") should contain. Based on
+the results, it either:
+
+1. Votes for an existing superblock trigger that matches the one calculated, or
+2. Creates and broadcasts a new superblock trigger if a matching one was not found
```

Comment:
```suggestion
2. Creates a new superblock trigger if a matching one was not found
```

AFAIK it only tries to broadcast if it's the winner.

https://github.com/dashpay/sentinel/blob/255f7cef7a1ad156edd66c287ca2cd039202de0f/bin/sentinel.py#L97-L100

Reply:
@nmarley Ah, yes. Any MN can vote, but only the winner can broadcast a new one?

Reply:
Yes, udjin's changes made it clearer than what I had here